contentcuration/contentcuration/static/js/edit_channel/router.js

```diff
@@ -10,7 +10,7 @@ var URL_CHAR_LIMIT = 7;
 var ChannelEditRouter  = Backbone.Router.extend({
   nodeCollection: new Models.ContentNodeCollection(),
   initialize: function(options) {
-    _.bindAll(this, "navigate_channel_home", "preview_page", "edit_page", "clipboard_page", "admin_page");
+    	_.bindAll(this, "navigate_channel_home", "preview_page", "edit_page", "clipboard_page");
```

Comment:
No longer using the `ChannelEditRouter` for the admin page!  The admin page has it's own router now :)